digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1604,11 +1600,18 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
             updateRoadwaysAndLinearLocationsWithProjectLinks(updatedStatus, projectID)
           }
         }
-        false
+        val roadNetworkDAO = new RoadNetworkDAO
+        if(roadNetworkDAO.hasCurrentNetworkErrors){
+          logger.error(s"Current network have errors")
+        } else {
+          val roadNumbers = projectLinkDAO.fetchRoadNumbersByProjectIdHistory(projectID)
```

Comment:
We can use updateRoadwaysAndLinearLocationsWithProjectLinks to return the road numbers that were used in the project instead of create another query just to find the roadnumbers in the project that is history now